index.html

```diff
@@ -1,9 +1,9 @@
-<!doctype html>
+<!Doctype html>
 <!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7" lang=""> <![endif]-->
 <!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8" lang=""> <![endif]-->
 <!--[if IE 8]>         <html class="no-js lt-ie9" lang=""> <![endif]-->
 <!--[if gt IE 8]><!-->
-<html class="no-js" lang="en">
+<html class="no-js" lang="">
```

Comment:
I would assume that screen readers would benefit from the language tag. Is there a reason it was removed?

Reply:
@profounddark I  must have removed it by accident when I was trying to figure out the which branch I was in.